joern-cli/frontends/jimple2cpg/src/main/scala/io/joern/jimple2cpg/Main.scala

```diff
@@ -20,8 +23,13 @@ private object Frontend {
 
   val cmdLineParser: OParser[Unit, Config] = {
     val builder = OParser.builder[Config]
-    import builder.programName
-    OParser.sequence(programName("jimple2cpg"))
+    import builder._
+    OParser.sequence(
+      programName("jimple2cpg"),
+      opt[String]("android")
+        .text("Optional path to android.jar while processing apk file.")
+        .action((android, config) => config.copy(android = Some(android)))
```

Comment:
```suggestion
        .action((android, config) => config.copy(android = Option(android)))
```
If you do `Some(null)` then it won't become a `None` so always safer to use the `Option` route.